drivers/temperature/adt7420/adt7420.h

```diff
@@ -54,7 +57,21 @@
 #define ADT7420_A1_PIN(x)	(((x) & 0x1) << 1) // I2C Serial Bus Address Selection Pin
 #define ADT7420_ADDRESS(x,y)	(0x48 + ADT7420_A1_PIN(x) + ADT7420_A0_PIN(y))
 
-/* ADT7420 registers */
+/* ADT7320  (SPI) registers */
+#define ADT7320_REG_STATUS		0x00 // Status
+#define ADT7320_REG_CONFIG	    0x01 // Configuration
+#define ADT7320_REG_TEMP    	    0x02 // Temperature value
+#define ADT7320_REG_ID		    0x03 // ID
+#define ADT7320_REG_T_CRIT	    0x04 // Temperature CRIT setpoint (147'C)
+#define ADT7320_REG_HIST	    0x05 // Temperature HYST setpoint (5'C)
+#define ADT7320_REG_T_HIGH    	    0x06 // Temperature HIGH setpoint (64'C)
+#define ADT7320_REG_T_LOW    	    0x07 // Temperature LOW setpoint (10'C)
+
+/* ADT7320 SPI command byte */
+#define ADT7320_WRITE_MASK_CMD      0b00111000 // SPI write command
+#define ADT7320_READ_CMD            0b01000000 // SPI read command
+
+/* ADT7420 (I2C) registers */
```

Comment:
Fix the indentation: Makes them be inline with the rest of the defines:
See this view: https://github.com/analogdevicesinc/no-OS/blob/72f0062ebdf4f831a346afa128166f29d62340c0/drivers/temperature/adt7420/adt7420.h?ts=8
How it is displayed here is not really accurate.